### DISCUSSION TITLE
Directus v9 SAML Integration

### DISCUSSION THREAD

**Discussion by auyongjinyoo** [directus/directus#16582] [Archive]
I have go through Directus Docs and found out it's saying support SSO SAML mechanisms. There are lots example for the OpenID and OAuth2.0 but I could not found any example shows how to configure the SAML.

Does anyone here have do the configuration and share the experience how to configure the SAML?

Thanks in advance.

**Comment by wfriedman89** [comment]
Some docs related to SAML config:  https://docs.directus.io/self-hosted/config-options.html#saml

**Reply by drpdishant** [reply]
@wfriedman89 the official docs don't provide any working example or guide to setup up one for saml

**Reply by rijkvanzanten** [reply]
> @wfriedman89 the official docs don't provide any working example or guide to setup up one for saml

Actual configuration for SAML is very different vendor to vendor, so a working example for something like Google wouldn't necessarily translate to any other vendor 👍🏻

**Reply by drpdishant** [reply]
Yes but an example with few vendors or opinionated providers can be helpful. Keycloak is good to start with.
